### PR TITLE
Add VRChat icon

### LINKED ISSUE #7763: Request: VRChat
### Brand Name

VRChat

### Website

https://vrchat.com

### Popularity Metric

The SimilarWeb rank is 8,292. See https://www.similarweb.com/de/website/vrchat.com

### Official Resources for Icon and Color

*Couldn't find an official SVG*

Brand Guidelines: [VRChat Partnership Branding_V2](https://docs.google.com/presentation/d/e/2PACX-1vRlm7h3Vh5sjRVS478amvSywI1D7GWuMzv_kYkxOPW8dELnAVJq9Fbiqp_hnE1o2N-v2NyKOntdiaOz/pub?slide=id.g7ca235226a_0_13)
Icon: [PNG](https://drive.google.com/file/d/1bgReLvOZxtujuU_FjWo7bPVxZ3FQf0zm/view)
Color: `FFFFFF` / `000000`

### Additional Comments

_No response_

### Contributing

No

### PR DESCRIPTION
**Pull request by jrhaberland**
vrchat

**Issue:** closes #7763

**Popularity metric:**
The SimilarWeb rank is 8,674. See https://www.similarweb.com/website/vrchat.com

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

**Comment by adamrusted** [comment]
Just for reference, in their brand guidelines they explicitly state you shouldn't alter the logos.
image

The example they give isn't representative of what we're doing on this project, but unless there is a 'single colour with transparent background' version in the press kit, we are altering their logo and will need permission to include.

**Comment by jrhaberland** [comment]
Yeah I used the one with the black outline. Since we are not altering the logo will we still need to ask for permission @adamrusted?

File: _data/simple-icons.json [hunk 1]
```diff
@@ -17044,6 +17044,12 @@
             "hex": "DA074A",
             "source": "https://commons.wikimedia.org/wiki/File:VOX_Logo_2013.svg"
         },
+        {
+            "title": "VRChat",
+            "hex": "000000",
+            "source": "https://drive.google.com/file/d/10BIsrBGt_4R7OQdYZIu6Nd5l1ogzOjVc/view?usp=drive_link",
+            "guidelines": "https://docs.google.com/presentation/d/e/2PACX-1vRlm7h3Vh5sjRVS478amvSywI1D7GWuMzv_kYkxOPW8dELnAVJq9Fbiqp_hnE1o2N-v2NyKOntdiaOz/pub?slide=id.g7ca235226a_0_13"
```
Comment (uncenter) [inline _data/simple-icons.json:17051]: As per https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines we should ideally link to a page where this Drive folder is *linked* to but not the Drive folder itself.

**Review by uncenter — CHANGES_REQUESTED** [review]

**Comment by uncenter** [comment]
> Yeah I used the one with the black outline. Since we are not altering the logo will we still need to ask for permission @adamrusted?

Yep we should be fine not asking here since there is that outlined version as Adam said.

File: _data/simple-icons.json (hunk 1 above)
Comment (jrhaberland) [reply _data/simple-icons.json]: That should take care of that

**Comment by jrhaberland** [comment]
> > Yeah I used the one with the black outline. Since we are not altering the logo will we still need to ask for permission @adamrusted?
> 
> Yep we should be fine not asking here since there is that outlined version as Adam said.

Sounds good, thanks for helping me

File: _data/simple-icons.json [hunk 2]
```diff
@@ -17044,6 +17044,12 @@
             "hex": "DA074A",
             "source": "https://commons.wikimedia.org/wiki/File:VOX_Logo_2013.svg"
         },
+        {
+            "title": "VRChat",
+            "hex": "000000",
+            "source": "https://hello.vrchat.com",
```
Comment (uncenter) [inline _data/simple-icons.json:17050]: Hmm I'm not seeing SVGs on https://hello.vrchat.com/. The link you have for `guidelines` has a link to the presskit as well, did you mean that?
```suggestion
            "source": "https://hello.vrchat.com/press",
```

Comment (uncenter) [reply _data/simple-icons.json:17050]: Icon is a perfect match for the logo file I found at that press link, I'll update it.

**Review by uncenter — APPROVED** [review]

**Pull request merged**